packages/shim-prompts/src/index.ts

```diff
@@ -27,7 +27,7 @@ export const prompt: typeof globalThis extends { prompt: infer T } ? T
     (globalThis as any)["prompt"] ??
       function prompt(
         message = "Prompt",
-        defaultValue = undefined,
```

Comment:
I'm pretty sure we don't need this since it would [default to `undefined`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#description) either way.